udata/core/reports/models.py

```diff
@@ -11,8 +11,38 @@
 
 from .constants import REPORT_REASONS_CHOICES, REPORTABLE_MODELS
 
+REPORT_STATUS_ONGOING = "ongoing"
+REPORT_STATUS_DONE = "done"
```

Comment:
Do we want to go with handle/unhandle wording?

Reply:
Switch to bool in 2b1e85f45